packages/dialtone-tokens/postcss/constants.cjs

```diff
@@ -0,0 +1,190 @@
+/**
+ * This file contains the constants that we use to dynamically generate
+ * the dialtone utility classes and variables.
+ *
+ * Put here the unique values that are needed to generate the classes or mappings
+ * to utilities/variables.
+
+ * This data shouldn't duplicate values under `/docs/_data/` folder
```

Comment:
Please update the comment to match the new location and usage

Reply:
done, also removed some consts that weren't used.